src/main/scala/mesosphere/marathon/raml/DefaultConversions.scala

```diff
@@ -37,4 +42,22 @@ trait DefaultConversions {
       case (k, v) => key.write(k) -> value.write(v)
     }
   }
+
+  implicit val timestampWrites: Writes[Timestamp, String] = Writes { _.toString }
+
+  implicit val pathIdWrites: Writes[PathId, String] = Writes { _.toString }
+
+  implicit val instanceIdWrites: Writes[Instance.Id, String] = Writes { _.toString }
+
+  implicit val taskStateWrites: Writes[mesos.TaskState, MesosTaskState] = Writes { taskState =>
+    val name = taskState.name()
+    MesosTaskState.fromString(name).getOrElse(throw new IllegalArgumentException(s"$name is an unknown Mesos task state"))
```

Comment:
The exception should not really happen since it's a write. What shall we do here?

Reply:
I think if it's something that shouldn't really happen then it's ok to use an exception.